styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -194,6 +197,29 @@ private void run() {
           }
           break;
 
+        case RESOURCE:
```

Comment:
I fully agree. I wanted to modularize the different CLI components but wanted to keep this PR lean. Let's do it next!